SpanishFilter/sections/specific.txt

```diff
@@ -4,6 +4,7 @@
 !
 segredosdomundo.r7.com###custom_html-6
 r7.com##.r7-flex-adv__placeholder
+vertele.eldiario.es##.captioned-info
```

Comment:
This rule hides images description, so I think it shouldn't be added.

Reply:
Ok I will delete it.